ui/src/app/app-routing.module.ts

```diff
@@ -6,11 +6,11 @@ import { ChangelogViewComponent } from './changelog/view/view';
 import { EdgeComponent } from './edge/edge.component';
 import { ChannelthresholdChartOverviewComponent } from './edge/history/channelthreshold/channelthresholdchartoverview/channelthresholdchartoverview.component';
 import { AutarchyChartOverviewComponent } from './edge/history/common/autarchy/overview/overview';
+import { OverviewComponent } from './edge/history/common/grid/overview/overview';
```

Comment:
This won't work for multiple imports with same name 'OverviewComponent'. You could use
`import { OverviewComponent as HistoryGridComponent } from './edge/history/common/grid/overview/overview';`

Reply:
thats true, will do it